### PR TITLE
HARP-8873 Removed phase loading

### PR DESCRIPTION
**Pull request by MPursche**
Signed-off-by: Marcel Pursche <marcel.pursche@here.com>

Thank you for contributing to harp.gl!

Before requesting a pull request, please remember to check the following documents:
* [contribution guidelines](https://github.com/heremaps/harp.gl/blob/master/CONTRIBUTING.md)
* [coding style](https://github.com/heremaps/harp.gl/blob/master/CODINGSTYLE.md)

If you are adding new functionality we would highly appreciate if you can describe what is the capability you are adding and even better if you can add some examples. Please also remember to add tests for it.

# CI Check

Our bots will check whether your PR can be directly integrated into the mainline. We have some internal integration tests running on the background, our bots will inform you of the next steps and someone from our team will take a look and help if needed!

And please do not forget to sign-off your commit! You can read more about DCO [here](https://julien.ponge.org/blog/developer-certificate-of-origin-versus-contributor-license-agreements/). But, in short, you just need to use `git commit -s` or append `--signoff` when you are committing to the repo.

Happy contributing!

### CODE REVIEW

**Comment by MPursche** [comment]
@harpgl-bot Test this please.

**Comment by MPursche** [comment]
@harpgl-bot ok to test.

**Comment by ninok** [comment]
@harpgl-bot ok to test

**Comment by ninok** [comment]
@heremaps-bot ok to test

File: @here/harp-mapview/lib/MapView.ts [hunk 1]
```diff
@@ -526,19 +525,6 @@ export interface MapViewOptions extends TextElementsRendererOptions {
      */
     maxFps?: number;
 
-    /**
```
Comment (ninok) [inline @here/harp-mapview/lib/MapView.ts]: Please add information about tghis breaking API change to the release note draft:
https://github.com/heremaps/harp.gl/releases/tag/untagged-b803742d56359e72466c

File: @here/harp-mapview/lib/MapView.ts [hunk 2]
```diff
@@ -983,10 +969,7 @@ export class MapView extends THREE.EventDispatcher {
             mapPassAntialiasSettings
         );
 
-        this.m_tileGeometryManager =
-            this.m_options.enablePhasedLoading === true
-                ? new PhasedTileGeometryManager(this)
-                : new SimpleTileGeometryManager(this);
+        this.m_tileGeometryManager = new SimpleTileGeometryManager(this);
```
Comment (ninok) [inline @here/harp-mapview/lib/MapView.ts]: What is this manager doing? Do we still need it? Maybe rename `SimpleTileGeometryManager` to `TileGeometryManager`

**Pull request merged**